tests-integration/tests/region_failover.rs

```diff
@@ -56,6 +75,16 @@ async fn test_region_failover() {
     let mut distribution = find_region_distribution(&cluster).await;
     info!("Find region distribution again: {distribution:?}");
 
+    // Waits for invalidating table cache
+    time::sleep(Duration::from_millis(100)).await;
```

Comment:
Can we assert that the table cache is invalidated?

Reply:
It's better to wait several hundred milliseconds. The invalidating table cache step only broadcasts the instructions to the frontend but doesn't wait for the response from the frontend.

Reply:
It doesn't matter the Frontend doesn't response the cache invalidation instruction. You have full control of Frontend instance in the testing, just retrive, downcast or expose whatever objects you want. It's better to assert the cache indeed existed before the region failover started, and the cache is gone after. There's feature flag in frontend crate called `testing` you can use.

Reply:
Yes, @MichaelScofield that's my point.